config/initializers/5_telemetry.rb

```diff
@@ -1,3 +1,11 @@
+# Set multiprocess-friendly data store
+require 'prometheus/client'
```

Comment:
Doesn't this break all installations where the telemetry group is unavailable?

Reply:
https://github.com/theforeman/foreman/pull/7271

Reply:
https://github.com/theforeman/foreman/pull/7271